### DISCUSSION TITLE
[blocks]: Removal of blocks

### DISCUSSION THREAD

**Discussion by joeypy** [shadcn-ui/ui#5457] [Blocks Request]
### Description

What happened to the other blocks and how to be able to isolate and select them dynamically? Will they no longer be shown in the documentation as before?

**Comment by MounirAia** [comment]
Yes, why are the old blocks disappeared?

**Comment by evmartinelli** [comment]
Why olders blocks are no longer available?

**Reply by MounirAia** [reply]
They are not available on the new version of the website. They are creating npm command to download them directly (from what I understood). But you have access to them on the old version of the website:

Website: https://ui-8dy8ykf4h-shadcn-pro.vercel.app/blocks

Discussion talking about the problem: https://github.com/shadcn-ui/ui/issues/5435
